core/controllers/domain_objects_validator.py

```diff
@@ -27,13 +27,13 @@
 from core.domain import collection_domain
 from core.domain import config_domain
 from core.domain import exp_domain
+from core.domain import state_domain
 import python_utils
 
-from typing import Any, Dict # isort:skip  pylint: disable=wrong-import-order, wrong-import-position, unused-import, import-only-modules
+from typing import Any, Dict, Optional, Union # isort:skip  pylint: disable=wrong-import-order, wrong-import-position, import-only-modules
```

Comment:
I think this should be possible.

```suggestion
from typing import Any, Dict, Optional, Union # isort:skip 
```

Reply:
Thank you, updated.
Please take a look.